force-app/main/default/classes/DMLOperations.cls

```diff
@@ -17,125 +17,152 @@
  *
- * @author Your Name
+ * @author Christopher Van Dusen
  */
 
 
 public with sharing class DMLOperations {
 
-    /**
+    /** Q1
      * Question 1
      * Create a new Account object and inserts it into the database.
      * The Account will have a random name for uniqueness.
      * @return The Id of the newly created Account.
      */
-    public static Id insertNewAccount() {
-        // Create a new Account instance
+        public static Id insertNewAccount() {
+            // Create a new Account instance
+            Account newAccount = new Account();
 
-        // Set some fields on the Account
+            // Set some fields on the Account
+            newAccount.Name = 'Builders Vision';
 
-        // Insert the Account into the database
+            // Insert the Account into the database
+            insert newAccount;
 
-        // Return the Id of the newly created Account
-        return null;
-    }
+            // Return the Id of the newly created Account
+            return newAccount.id;
+        }
 
-    /**
+    /** Q2
      * Question 2
      * Create a new Account with the specified name and industry provided.
      * @param name The name of the account to be created.
      * @param industry The industry of the account to be created.
      */
-    public static void createAccount(String name, String industry) {
-        // Create a new Account and set the fields inline
+        public static void createAccount(String name, String industry) {
+            // Create a new Account and set the fields inline
+            Account newAccount = new Account(Name = name, Industry = industry);
 
-        // Insert the Account
-    }
+            // Insert the Account
+            insert newAccount;
+        }
 
-    
-    /**
+    /** Q3
      * Question 3
      * Create a new Contact and link it to the Account id provided then inserts it into the database.
      * @param accountId The Id of the Account to link the Contact to.
      * @return The Id of the newly created Contact.
      */
-    public static Id insertNewContact(Id accountId) {
-        // Create a new Contact instance
+        public static Id insertNewContact(Id accountId) {
+            // Create a new Contact instance
+            Contact newContact = new Contact();
+
+            // Set some fields on the Contact
+            newContact.FirstName = 'Christopher';
+            newContact.LastName = 'Van Dusen';
 
-        // Set some fields on the Contact
-        // Set the AccountId field to the given accountId
+            // Set the AccountId field to the given accountId
+            newContact.AccountId = accountId;
 
-        // Insert the Contact into the database
+            // Insert the Contact into the database
+            insert newContact;
 
-        // Return the Id of the newly created Contact
-        return null;
-    }
+            // Return the Id of the newly created Contact
+            return newContact.id;
```

Comment:
Just a heads up that sObject field names should always start with a capital letter:
```suggestion
            return newContact.Id;
```